internal/store/deployment.go

```diff
@@ -122,6 +122,30 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_deployment_status_condition",
+			Type: metric.Gauge,
+			Help: "The current status conditions of a deployment.",
```

Comment:
Only nit I would say we remove the `.` at the end. :)

Reply:
All the other help texts have periods.